src/System.Net.Sockets/src/System/Net/Sockets/_ReceiveMessageOverlappedAsyncResult.cs

```diff
@@ -10,200 +10,56 @@
 
 namespace System.Net.Sockets
 {
-    //
```

Comment:
I think that we covered this in a previous PR: the current state of the code does not exactly reflect the shape of the eventual factoring. Over time, the shape of this code will move towards what was discussed in #10.